tests/telemetry/test_application.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+from ddtrace.internal.telemetry.data.application import APPLICATION
+from ddtrace.internal.telemetry.data.application import Application
+from ddtrace.internal.telemetry.data.application import format_version_info
+from ddtrace.internal.telemetry.data.application import get_version
+
+
+def test_application():
```

Comment:
docstring would be nice, stating that this is testing the default value of `APPLICATION` without any env variables set

Reply:
same for all other test cases